.travis.yml

```diff
@@ -3,9 +3,15 @@ rvm:
   - 2.2
   - 2.3
   - 2.4.1
+  - 2.5.1
   - jruby
   - rbx-2
 
+gemfile:
+  - gemfiles/ruby20.gemfile
```

Comment:
I dont think there is any value keeping this file, for sidekiq 4 and nokogiri 1.6.x ?!?